sdk/storage/inc/common/storage_error.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <map>
+#include <stdexcept>
+#include <string>
+
+namespace Azure { namespace Core { namespace Http {
+  class Response;
```

Comment:
Forward declaration is not recommended to use. Ref https://google.github.io/styleguide/cppguide.html#Forward_Declarations